### ISSUE TITLE
DistributionNotFound error running pywal

### ISSUE THREAD

**Issue by yaabdalla** [dylanaraps/pywal#159]
```
Traceback (most recent call last):
  File "/usr/bin/wal", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3138, in <module>
    @_call_aside
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3122, in _call_aside
    f(*args, **kwargs)
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3151, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 664, in _build_master
    ws.require(__requires__)
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 981, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/nimbus/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 867, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pywal==1.1.0' distribution was not found and is required by the application
```

Getting this error. I have Python 3.6 installed via Anaconda. I installed pywal via the python-pywal-git AUR repo. On Manjaro 17.1.0.

**Comment by dylanaraps** [comment]
This is a problem with your environment and not `pywal`.

**Comment by yaabdalla** [comment]
Yeah, I realized this just recently trying to install and set up mopidy. My situation and solution for thos who come across the same issue:

My whole python set up is borked. I recently installed Anaconda with Python 3.6.3 because Anaconda is great. But Anaconda has its own packages, its own version of pip. But it symlinked python and pip to its own python and pip.

So on my machine, I now have 3 versions of Python. Python2.7, Python 3.6.4, and Python3.6.3(Anaconda). Each with their own pip install. The package from AUR installed pywal with pip3, so it was actually under Python 3.6.4. I tried installing pywal under the Anaconda env, but then I couldn't figure out how to call wal from CLI. So my other solution was to keep pywal installed as it is, under the system Python3. **I then edited the wal file in` /usr/bin/`, changing `#!/usr/bin/python` to `#!/usr/bin/python3`.** This got around python being aliased with the anaconda python.

Gonna have to figure this mess out some time, but not today lol.
